DTDL/base_options.json

```diff
@@ -4,5 +4,13 @@
   "@type": "Interface",
   "displayName": "Options base",
   "contents": [
+    {
+      "@type": "Property",
+      "name": "Name",
+      "schema": "string",
+      "writable": true,
+      "displayName": "Name",
+      "description": "Human-readable name of the source."
+    }
```

Comment:
Human-readable name of the source? Why it's called `Name` and why it's necessary at all?

Reply:
It's used to give the User the cabapility to choose a Name other than the Standard Display Name like (Beckhoff TwinCat ADS Parameter). For example if the User have the Requirement to set ten Parameters, he would have an List with Parameters which have all the same Name. "Beckhoff TwinCat ADS Parameter" x 10. 

I thought it would be a nice to have for the User to let him decide the Naming of the Parameters and Options ?

Reply:
Understood, thank you. Can we change the description to simply "Human readable name"? Because it's not a must that the source will be part of it. I agree, mostly it will, but it's not a must.

Reply:
Changed to Human-readable Name